backend/src/render/common.mjs

```diff
@@ -0,0 +1,15 @@
+import {kB, TB} from "../util.mjs";
+
+/**
+ * @param {UsageSummary} table
+ * @returns {string}
+ */
+export const scyllaPricingUrl = (table) => {
```

Comment:
`table` isn't really a good parameter name here, as you call it from the whole summary description. I particularly got confused.

Reply:
Fixed in [bb1c237](https://github.com/scylladb/dynamodb-pricing-comparison/pull/8/commits/bb1c2370da0e87f9b55c0c9e60c5fa055e77f423).